driver/modern_bpf/helpers/extract/extract_from_kernel.h

```diff
@@ -167,6 +157,22 @@ static __always_inline void extract__ino_from_inode(struct inode *f_inode, u64 *
 	BPF_CORE_READ_INTO(ino, f_inode, i_ino);
 }
 
+/**
+ * @brief Return epoch in ns from struct timespec64.
+ *
+ * @param time timespec64 struct.
+ * @return epoch in ns.
+ */
+static __always_inline u64 extract__epoch_ns_from_time(struct timespec64 time)
+{
+	time64_t tv_sec = time.tv_sec;
+	if (tv_sec > 0)
+	{
+		return (tv_sec * (uint64_t) 1000000000 + time.tv_nsec);
+	}
+	return 0;
```

Comment:
Hi @incertum , I wonder whether `return time.tv_nsec` should be used here? Or are there any reasons returning time starting with 1 sec?

Reply:
Nope just messed up a bit :) fixed it.